scripts/utils.sh

```diff
@@ -20,15 +12,6 @@ function setGrep() {
     fi
 }
 
-# Set ARCH env so as a container executes without issues in a portable way
-# Should be amd64 for linux/macos x86 hosts, and arm64 for macos M1
-# It is referenced in the docker composer yaml, and then passed as a Dockerfile ARG
-function setARCH() {
-    export ARCH=$(uname -m)
-    [ $ARCH = x86_64 ] && ARCH=amd64
-    echo "ARCH set to $ARCH"
-}
-
```

Comment:
`setGrep` and`convertToLower` need `setARCH`. I'd be happy to get rid of it all though

Reply:
addressed in https://github.com/ava-labs/teleporter/pull/393/commits/1a083a23896dbfca4faf020bc667837ffa089e1b